kit/popups/src/useWindowDimensions.web.ts

```diff
@@ -0,0 +1,11 @@
+import { useDimensions } from '@tonlabs/uicast.keyboard';
```

Comment:
What do you think of moving this hook into another package? It doesn't look like it should belong to the .keyboard

Reply:
I agree. Done.